backend/btrixcloud/main_migrations.py

```diff
@@ -0,0 +1,62 @@
+"""entrypoint module for init_container, handles db migration"""
+
+import os
+import sys
+import asyncio
+
+from .ops import init_ops
+from .db import update_and_prepare_db
+
+
+# ============================================================================
+# pylint: disable=too-many-function-args, duplicate-code
+async def main() -> int:
+    """init migrations"""
+
+    # pylint: disable=import-outside-toplevel
+    if not os.environ.get("KUBERNETES_SERVICE_HOST"):
+        print(
+            "Sorry, the Browsertrix Backend must be run inside a Kubernetes environment.\
+             Kubernetes not detected (KUBERNETES_SERVICE_HOST is not set), Exiting"
+        )
+        return 1
+
+    (
+        org_ops,
+        crawl_config_ops,
+        _,
+        crawl_ops,
+        _,
+        page_ops,
+        coll_ops,
+        _,
+        storage_ops,
+        background_job_ops,
+        _,
+        user_manager,
+        invite_ops,
+        _,
+        mdb,
+    ) = init_ops()
+
+    await update_and_prepare_db(
+        mdb,
+        user_manager,
+        org_ops,
+        crawl_ops,
+        crawl_config_ops,
+        coll_ops,
+        invite_ops,
+        storage_ops,
+        page_ops,
+        background_job_ops,
+        {},
```

Comment:
I think we can/should remove this (and uses of `db_inited` inside `update_and_prepare_db`) following these changes, since it's not being used for anything in the init container and `await_db_and_migrations` will now set `db_inited` for main separately.

Reply:
Removed!